documentation/README.md

```diff
@@ -22,6 +22,17 @@ This command starts a local development server and opens up a browser window. Mo
 $ npm run build
 ```
 
-This command generates static content into the `build` directory and can be served using any static contents hosting service.
+This command generates static content into the `build` directory and can be served using any static contents hosting service. 
 
+### Testing your Build Locally
 
+It is important to test your build locally before deploying it for production. Docusaurus provides a docusaurus serve command for that:
+
+```bash
+npm run serve
+```
+
+By default, this will load your site at http://localhost:3000/.
```

Comment:
once you hardcoded the port to 3030, then you need to update this line -> 3030 accordingly.
p/s I believe the reason why we don't want to use the default port 3000 in this documentation site is because this port is taken over by the demo explorer, and the URL to that app local is mentioned a lot throughout the documentation site

Reply:
Yes, thank you. I'll update the readme file.